src/ssl_stubs.c

```diff
@@ -1828,7 +1840,7 @@ CAMLprim value ocaml_ssl_flush(value socket)
     if (ret != 1) {
       caml_acquire_runtime_system();
       caml_raise_with_arg(*caml_named_value("ssl_exn_flush_error"),
-			  Val_bool(ret==-1));
+			  Val_bool(BIO_should_retry(bio)));
```

Comment:
can these be a separate PR? I think I have a different interpretation of the docs, so let's discuss there.

Reply:
OK, I found the doc not really clear on that point (and a few other too)

Reply:
Done, this is PR #120.